spec/helpers/sufia_helper_spec.rb

```diff
@@ -1,3 +1,7 @@
+def new_state
```

Comment:
I agree that would be better, but it would change the semantics where `new_state` is invoked twice in one test.  It was written as a method previously, and there are only so many turds I care to polish as ancillary work to my actual purpose.  Previously it was buried in the middle of a test definition, I pulled it out so that people would notice it and say, as you have, "hey, that doesn't belong there!"  But I'm not entertaining fixing that now.

There are similar instances in various test files, btw.  If you want to ticket improving the rspec conventions of the set, I could conceivably work on them as a group.